lib/dotcom/utils/date_time.ex

```diff
@@ -0,0 +1,54 @@
+defmodule Dotcom.Utils.DateTime do
+  @moduledoc """
+  A collection of functions for working with date_times.
+
+  Consuming modules are responsible for parsing or converting date_times.
+  They should *always* call `coerce_ambiguous_time/1` before using a date_time.
+  This is mainly because Timex has so many functions that serve as entry points to date_times.
+  Those functions can return ambiguous date_times during DST transitions.
+  """
+
+  require Logger
+
+  use Timex
+
+  @typedoc """
+  A range of time that can be open ended on either side.
+  """
+  @type time_range() :: {DateTime.t() | nil, DateTime.t() | nil}
```

Comment:
Naming: Should this be named `date_time_range()` instead of `time_range()`? When I was looking at the typespecs for the `service_*_range` functions, I was wondering why they were dropping the date part.

Although... per [this comment](https://github.com/mbta/dotcom/pull/2372/files#r1949895730), possibly a better name would be `Utils.DateTimeRange.t()` 🙂

Reply:
Oh, and if this _isn't_ extracted to a separate module, I think it would make sense to have this type defined in the same module where `in_range?` is implemented, whether that's moving `in_range?` to this module or moving this type to the other module.

Reply:
changed to `date_time_range()`